### ISSUE TITLE
Supports health checking

### ISSUE THREAD

**Issue by MichaelScofield** [GreptimeTeam/greptimedb-ingester-go#44]
As https://github.com/GreptimeTeam/greptimedb/issues/4674 suggests, a method like `is_connected` is good to have in our Go SDK.

**Comment by killme2008** [comment]
We already have the heath checking protocol

https://github.com/GreptimeTeam/greptime-proto/blob/973f49cde88a582fb65755cc572ebcf6fb93ccf7/proto/greptime/v1/health.proto#L26

**Comment by daviderli614** [comment]
@killme2008 @MichaelScofield 
In the greptime healthcheck proto, the message returned by [HealthCheckResponse](https://github.com/GreptimeTeam/greptime-proto/blob/973f49cde88a582fb65755cc572ebcf6fb93ccf7/proto/greptime/v1/health.proto#L29C9-L29C28) is empty. It can only determine whether the network is connected. It should return a judgment similar to **ready** or **ok** field?

I added a [PR](https://github.com/GreptimeTeam/greptimedb-ingester-go/pull/45) and here are the results:
- The GreptimeDB running:
```
go run main.go
2024/09/11 01:54:47 health check response:
```
- The GreptimeDB stop:
```
go run main.go 
2024/09/11 01:54:38 rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp 127.0.0.1:4001: connect: connection refused"
2024/09/11 01:54:38 health check response: <nil>

```

**Comment by MichaelScofield** [comment]
@daviderli614 It's expected that the response body is empty. When the response is successfully returned, it means the server's connectivity is good, and that's all we need (fow now).

**Comment by daviderli614** [comment]
> @daviderli614 It's expected that the response body is empty. When the response is successfully returned, it means the server's connectivity is good, and that's all we need (fow now).

ok, PTAL: https://github.com/GreptimeTeam/greptimedb-ingester-go/pull/45

**Comment by MichaelScofield** [comment]
@daviderli614 I've merged #45 , please release it and don't forget to update our docs.

**Comment by daviderli614** [comment]
https://github.com/GreptimeTeam/greptimedb-ingester-go/pull/48
After this is merged, can write health check and hints docs together.
